common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/TestUtils.java

```diff
@@ -172,4 +170,54 @@ public static void assertIsSerializable(@NotNull Serializable object) {
       }
     }
   }
+
+
+  /**
+   * Asserts that the given {@link ThrowingRunnable} throws an exception of type {@code
+   * expectedThrowable} when executed.
+   *
+   * @param expectedThrowable the class of the exception that should be thrown
+   * @param runnable the {@link ThrowingRunnable} that should throw
+   * @param <T> the type of exception that should be thrown
+   * @return the thrown exception of type {@code expectedThrowable}
+   * @throws AssertionError if the given {@link ThrowingRunnable} throws a different type of
+   *     exception or does not throw at all
+   */
+  public static <T extends Throwable> T expectThrows(
```

Comment:
Is this a direct copy of what we have internally?

can't we use `assertThrows` directly provided by junit? (also I think that expectThrows was deprecated in favor of assertThrows).